src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -11,3 +11,5 @@ databaseChangeLog:
       file: db/changelog/changes/05-add-roles-and-users.yaml
   - include:
       file: db/changelog/changes/06-create-categories-and-books_categories-tables.yaml
+  - include:
+      file: db/changelog/changes/07-create-ShoppingCart-and-CartItem-tables.yaml
```

Comment:
naming is a bit inconsistent, let's use lowercase with underscore, as in your previous files